knowledge-base/error-table-minimum-size-designer.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Error 'Each table row and column maintains a minimum size, resulting in a minimum table size of {Width=10px, Height=10px}' in Report Designer
+description: Error 'Each table row and column maintains a minimum size, resulting in a minimum table size of {Width=10px, Height=10px}' in Report Designer.
+type: how-to
+page_title: Error 'Each table row and column maintains a minimum size, resulting in a minimum table size of {Width=10px, Height=10px}' in Report Designer
+slug: error-table-minimum-size-designer
+res_type: kb
+---
+
+## Description
+
```

Comment:
Describe when and where the error occurs. Give use cases and scenarios on its presence.

Also add an Error section with the whole error text in it after the description section.